java-checks-test-sources/src/main/java/checks/regex/GraphemeClustersInClassesCheck.java

```diff
@@ -0,0 +1,28 @@
+package checks.regex;
+
+import java.util.regex.Pattern;
+
+public class GraphemeClustersInClassesCheck {
+
+  void noncompliant(String str) {
+    Pattern.compile("aaa[è]aaa"); // Noncompliant [[sc=25;ec=29]] {{Extract these Grapheme Clusters from the character class.}}
```

Comment:
The issue message is in the plural, while there is apparently only one cluster, I feel it is strange
 Why not reporting directly on it and use a singular form?